server/sonar-server/src/main/java/org/sonar/server/component/ws/SearchProjectsAction.java

```diff
@@ -178,7 +177,7 @@ public void define(WebService.NewController context) {
       .setDescription("Sort projects by numeric metric key, quality gate status (using '%s'), last analysis date (using '%s'), or by project name.<br/>" +
         "See '%s' parameter description for the possible metric values", ALERT_STATUS_KEY, SORT_BY_LAST_ANALYSIS_DATE, PARAM_FILTER)
```

Comment:
Moreover, I think the list of allowed metric keys that can be used in the 'filter' parameter should be set into the description (line 146)